packages/core/src/lib/LinkTo/LinkTo.tsx

```diff
@@ -11,6 +12,8 @@ const LinkTo: React.FC<LinkToProps> = ({
     iconLayout = 'leading',
     text,
     type = 'external',
+    underline = true,
+    textColor="",
```

Comment:
for text color you do not need to set a default of blank string
`textColor` is better

Reply:
`underline = true,` is good however as the previous behaviour was true so it stops any unexpected behaviour